vuepress/docs/next/docs/curate/bundle-details.md

```diff
@@ -2,570 +2,264 @@
+
+```json
+{
+  "name": "my-bundle-name",
+  "description": "my bundle description",
+  "type" : "bundle",
+  "version": "0.0.1"
+  "svc": [
+        "keycloak"
+    ],
+}
+```
+
+### Microservices Specifications
+|Name|Type|Required|Possible Values|Description|
+|:-|:-|:-|:-|:------------------------|
+|`name`|String|Yes||Microservice name|
+|`stack`|Enum|Yes|*spring-boot   *node|Microservice stack |
+|`dbms`|Enum|No|*none  *embedded  *postgresql  *mysql  *oracle|DBMS required by the MS to provide services|
+|`ingressPath`|String|No||Custom ingress path|
+|`healthCheckPath`|String|No||Endpoint for a health check|
+|`deploymentBaseName`|String|No||Used for defining custom pod names|
+|`roles`|String[]|No||Exposed security roles|
+|`env`|[EnvironmentVariable[]](#environment-variables-specification)|No||Required environment variables|
+|`commands`|Command[]|No||Custom commands definitions|
+
+#### Microservices Sample Code
+```json
+"microservices": [
+    {
+      "name": "my-ms",
+      "stack": "spring-boot",
+      "dbms": "mysql",
+      "ingressPath": "/ingress",
+      "healthCheckPath": "/management/health",
+      "roles": ["admin"],
+      "env": [{ "VAR_1": "value" }]
+    }
+  ],
+```
+
+### Micro Frontends Specifications
+|Name|Type|Required|Possible Values|Description|
+|:-|:-|:-|:-|:------------------------|
+|`name`|String|Yes||Micro frontend name|
+|`stack`|Enum|Yes|*react   *angular|MFE stack|
+|`type`|Enum|Yes|*widget  *widget-config  *app-builder|Type of MFE|
+|`slot`|Enum|Yes for `type=app-builder`|*primary-header  *primary-menu  *content|Named reference to an App Builder embedded position in a specific layout|
+|`paths`|String[]|Yes for `type=app-builder` and `slot=content`||App Builder activation paths|
+|`titles`|String[]|Yes for `type=widget`||Localized widget labels|
+|`group`|String|Yes||Visibility group name|
+|`publicFolder`|String|No|Default is `public`|MFE public folder (typically where index.html is located)|
+|`apiClaims`|String[]|No||See [API Claim spec](#api-claim-specification) below|
+|`nav`|[MenuEntry[]](#menuentry-specification)|No||Bundle menu global links|
+|`commands`|Command[]|No||Custom commands definitions|
+|`buildFolder`|String|No|Default is `build`|Corresponds to the MFE build folder |
+|`mfeConfig`| MfeConfig[]|No||Custom configuration widget for App Builder|
+
+#### Micro Frontends Sample Code 
+```json
+ "microfrontends": [
+    {
+      "name": "my-mfe",
+      "stack": "react",
+      "titles": { "en": "my_en_mfe_title", "it": "my_it_mfe_title" }
+      "type": "app-builder",
+      "slot": "content",
+      "paths": ["/path1"],
+      "group" : "free",
+      "apiClaims": [...],
+    },
+    {
+      "name": "my-mfe2",
+      "stack": "react",
+      "type": "widget",
+      "publicFolder": "public",
+      "titles": { "en": "my_en_mfe2_title", "it": "my_it_mfe2_title" },
+      "group": "free",
+      "commands": { "build": "custom-command" }
+    }
+```
+#### mfeConfig Specifications
```

Comment:
Something is wrong here. mfeConfig isn't a thing. configMfe is a string indicating the custom element for the corresponding config MFE.  And what is listed in the mfeConfig section below should just be inline in the microfrontend section. You cannot have an array of mfeConfig containing params, contextParams, systemParams, you can just have the actual params, contextParams, and systemParams under the microfrontend.